src/components/DirectionalArrow.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import styled from 'styled-components'
+
+import arrowDownSvg from '../images/arrow_down.svg'
+import { TransactionDirection } from '../utils/transactions'
+
+interface Props {
+  direction: TransactionDirection
+}
+
+const DirectionalArrow = ({ direction }: Props) => {
+  const isReceiving = direction === 'in'
+  const color = {
+    circle: isReceiving ? 'rgba(108, 217, 158, 0.11)' : 'rgba(153, 153, 153, 0.11)',
```

Comment:
We should base those colors on theme values here (font secondary, "valid" color). 
We can use the `tinycolor` library to tweak the opacity of those values.